janusgraph-dist/src/assembly/static/conf/gremlin-server/gremlin-server-berkeleyje.yaml

```diff
@@ -14,7 +14,7 @@
 
 host: 0.0.0.0
 port: 8182
-scriptEvaluationTimeout: 30000
+evaluationTimeout: 30000
```

Comment:
Should mention this change in the changelog?

Reply:
Sure, make sense. I have made the change locally and will defer pushing it for some time with the hope that I could get more reviews. tp-tests take too much time and I don't want to waste much CI build cycles.